travis_configure.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+# Script for setting up OS specific CI test environment on travis-ci.org
+#
+# Copyright (C) 2016 Andreas Bok Andersen <andreas.bok@sociomantic.com>
+#
+
+set -e
+
+# Script which is run before installation script is called
+#
+# Installs fpm needed for deb packaging
+#
+# Returns 0
+function before_install {
+    if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+        if [[ "$DIST_PACKAGE_TARGET" == "DEB" ]]; then
+            gem install fpm
+        fi
+    fi
+
+    return 0
+}
+
+# Enable binary logging by setting log_bin and server_id in the [mysqld]
+# section of the mysql config file
+#
+# $1 path to mysql config file
+#
+# Returns 0
+function enable_mysqlbinlog
+{
+    sudo sed -i -e 's/^# *server.id.*$/server_id=1/g' $1
+    sudo sed -i -e 's/^# *log.bin.*$/log_bin=mysql-bin/g' $1
```

Comment:
Nope, sudo is required to write to the root folder `/etc`. it is required for calling `service`